### PR TITLE
Fix: Benchmarking cleanup, deleted some files and removed old action

### PR DESCRIPTION
**Pull request by nravic**
## Checklist before requesting a review
- [ ] Have I performed a self-review?
- [ ] Have I added regression or unit tests (where it makes sense) for my changes?
- [ ] Have I updated the README if changes have resulted in it being out of date?
- [ ] Have I checked to ensure my PR only introduces the changes it's intended to? E.g. No extraneous formatting changes.
- [ ] Have I ensured that there are no performance regressions by checking benchmark results?
- [ ] Is this a breaking change? If yes, please describe areas affected and request reviews from developer stakeholders.

### CODE REVIEW

File: scripts/ci/helpers.sh [hunk 1]
```diff
@@ -127,9 +127,6 @@ setup_ci() {
     mkdir -p $HOME/.cedana
     echo '{"client":{"leave_running":false, "task":""}, "connection":{"cedana_url": "https://ci.cedana.ai"}}' > $HOME/.cedana/client_config.json
 
-    # Install recvtty
-    go install github.com/opencontainers/runc/contrib/cmd/recvtty@latest
```
Comment (yashanand1910) [inline scripts/ci/helpers.sh:131]: Why was this removed?

**Review by yashanand1910 — COMMENTED** [review]
Check comments.

File: scripts/ci/helpers.sh (hunk 1 above)
Comment (nravic) [reply scripts/ci/helpers.sh:131]: https://github.com/cedana/cedana/pull/385/commits/537d1db34ed813bd44f476c3c7ec8444137abb66

Added recvtty as a debug command in cedana so we wouldn't have to deal w/ downloading it anymore
